### PR TITLE
PP-11279: Run the connector-adminusers messaging pact test

### PR DESCRIPTION
**Pull request by oswaldquek**
This is safe to merge now as there is no published pact where the provider is adminusers and consumer is connector tagged with "master" and above (i.e. "test-fargate", "staging-fargate" etc). For reference the pact to be published is at https://github.com/alphagov/pay-connector/pull/4837.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -323,15 +323,15 @@
...
-            <artifactId>pact-jvm-consumer-junit_2.12</artifactId>
-            <version>${pact.version}</version>
+            <artifactId>pact-jvm-provider-junit</artifactId>
+            <version>4.0.10</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>xerces</groupId>
+                    <artifactId>xercesImpl</artifactId>
+                </exclusion>
```
[the remark this answers is not in the document]
Comment (oswaldquek) [reply pom.xml:333]: I copied it from [connector](https://github.com/alphagov/pay-connector/blob/master/pom.xml#L394-L405) but I can certainly remove it and see if the build still works.

[the remark this answers is not in the document]
Comment (oswaldquek) [reply pom.xml:333]: I guess I'm trying to be speedy as this piece of work is starling work and I'm officially off it and on to the openbanking team.

File: pom.xml [hunk 2]
```diff
@@ -323,15 +323,15 @@
...
-            <artifactId>pact-jvm-provider-junit_2.12</artifactId>
-            <version>${pact.version}</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>au.com.dius</groupId>
-            <artifactId>pact-jvm-consumer-junit_2.12</artifactId>
-            <version>${pact.version}</version>
+            <artifactId>pact-jvm-provider-junit</artifactId>
+            <version>4.0.10</version>
```
[the remark this answers is not in the document]
Comment (oswaldquek) [reply pom.xml:327]: [It's still being used](https://github.com/alphagov/pay-adminusers/blob/master/pom.xml#L366).

[the remark this answers is not in the document]
Comment (alexbishop1) [reply pom.xml:327]: Looks like it’s only used in one place — can we remove the variable and just actual value where the variable used to be used?

[the remark this answers is not in the document]
Comment (oswaldquek) [reply pom.xml:327]: I kind of think we should keep the `<pact.version>` property as we'll probably want to upgrade everything to 4.0.10 someday.

[the remark this answers is not in the document]
Comment (alexbishop1) [reply pom.xml:327]: I would say that’s the time to reintroduce the variable!

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**